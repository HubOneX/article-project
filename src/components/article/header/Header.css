.article-header {
  padding: 0px 30px;

  & .article-title {
    margin: 5px 0 15px;
    font-size: 32px;
  }

  @media screen and (max-width: 480px) {
    padding: 0px 20px;

    & .article-title {
      font-size: 25px;
    }
  }
}
