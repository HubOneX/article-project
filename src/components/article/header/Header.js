import Title from "./title/Title";
import ArticleInfo from "./articleInfo/ArticleInfo";
import "./Header.css";

const Header = ({ heading, publicationDate, author }) => {
  return (
    <div className="article-header">
      <ArticleInfo date={publicationDate} author={author} />
      <hr />
      <Title text={heading} />
    </div>
  );
};

export default Header;
