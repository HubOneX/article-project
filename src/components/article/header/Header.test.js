import renderer from "react-test-renderer";
import Header from "./Header";

describe("Header", () => {
  it("should match the snapshot", () => {
    const header = renderer.create(
      <Header
        heading="Heading!"
        publicationDate="2020-09-03T07:27:59.490Z"
        author="Test Name"
      />
    );

    expect(header).toMatchSnapshot();
  });
});
