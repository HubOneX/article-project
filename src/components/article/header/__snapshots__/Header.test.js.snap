// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should match the snapshot 1`] = `
<div
  className="article-header"
>
  <div
    className="article-info"
  >
    <p
      className="article-author"
    >
      Test Name
    </p>
    <p
      className="article-date"
    >
      09:27 Sep 3, 2020
    </p>
  </div>
  <hr />
  <h1
    className="article-title"
  >
    Heading!
  </h1>
</div>
`;
